daemon/volumes.go

```diff
@@ -333,7 +334,7 @@ func (container *Container) setupMounts() error {
 		mounts = append(mounts, execdriver.Mount{
 			Source:      container.Volumes[path],
 			Destination: path,
-			Writable:    container.VolumesRW[path],
+			Mode:        container.VolumesMode[path],
```

Comment:
Instead of passing these mode values directly (and subsequently pulling in github.com/docker/volumes into libcontainer to parse, maybe we can split this out, use "Writable", "LabelType" (shared/unshared/nil)